### PR TITLE
minor change for 'Get started with dataflow pipeline components'

### PR DESCRIPTION
**Pull request by ashakella**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [x] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/ml_ops/stage3/get_started_with_dataflow_pipeline_components.ipynb [hunk 1]
```diff
@@ -40,9 +40,17 @@
         "  </td>\n",
```
Comment (andrewferlitsch) [inline notebooks/community/ml_ops/stage3/get_started_with_dataflow_pipeline_components.ipynb:40]: <p>remove this bullet list:</p><p> </p><ul><li>Dataset</li><li>Pipeline</li><li>Model</li><li>Endpoint</li><li>AutoML Training Job</li><li>Batch Job</li><li>Custom Job</li><li>Hyperparameter Tuning Job</li><li>Cloud Storage Bucket</li></ul><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Comment (ashakella) [reply notebooks/community/ml_ops/stage3/get_started_with_dataflow_pipeline_components.ipynb:40]: @andrewferlitsch : Correct the list of cleanup items.

**Review by andrewferlitsch — APPROVED** [review]

**Comment by ashakella** [comment]
@andrewferlitsch : Manually tested on colab and workbench.

**Comment by andrewferlitsch** [comment]
Automated test not ran. Submitter manually confirmed execution with Colab/Workbench. I am manually merging the PR.

**Pull request merged**